Scoop/SearchRidesViewController.swift

```diff
@@ -249,6 +250,15 @@ class SearchRidesViewController: UIViewController {
             }
         }
     }
+    
+    @objc func textFieldDidChange(sender: UITextField) {
+        guard let text1 = departureTextField.textField.text, !text1.isEmpty, let text2 = arrivalTextField.textField.text, !text2.isEmpty, let text3 = departureDateTextField.text, !text3.isEmpty else {
+            findTripsButton.backgroundColor = .disabledGreen
+            return
+        }
```

Comment:
Can you separate these into different lines? Add a new line after every comma. The `let` should be lined up and indented.